apps/i18n/applab/en_us.json

```diff
@@ -511,11 +512,11 @@
   "makerSetupUpdateMBFirmata": "Update micro:bit",
   "makerSetupUpdateMBFirmataDescription": "Update micro:bit with latest version of Firmata",
   "makerSetupVerifyComponents": "Oh no! Something unexpected went wrong while verifying the board components.",
-  "makerSetupWebSerialConnectToComputer": "Connect the board to your computer.",
+  "makerSetupWebSerialConnectToComputer": "Connect the board to your computer using a USB cable.",
   "makerSetupWebSerialConnectToBoardButton": "Press the “Connect to Board” button below.",
-  "makerSetupWebSerialWindowConnect": "A window should pop up with a row that includes \"Circuit Playground\" or \"micro:bit\". Select that row, then click \"Connect\".<br>If you are connecting a Circuit Playground Express, and 'No compatible devices found.' appears in the window, please install the required Firmata firmware - instructions [here](https://learn.adafruit.com/adafruit-circuit-playground-express/code-org-csd)",
+  "makerSetupWebSerialWindowConnect": "A window should pop up with a row that includes \"Circuit Playground\" or \"micro:bit\". Select that row, then click \"Connect\".<br>⚠️ If you are connecting a Circuit Playground Express, and 'No compatible devices found.' appears in the window, please install the required Firmata firmware - instructions [here](https://learn.adafruit.com/adafruit-circuit-playground-express/code-org-csd)",
```

Comment:
Do we need to use &#9888; for the warning emoji in html?

Reply:
🤦‍♀️ `&#9888;`

Reply:
It looks like this is an established practice already to put emojis instead of their html codes! so nevermind!

Reply:
We already use emojis in validation strings that are rendered in html. Here's an example: https://github.com/code-dot-org/code-dot-org/blob/899dd4c60ffe64b45979a9f7c1be659250829747/apps/i18n/dance/en_us.json#L3